Bottom: a32334e7c7b624a17584a5528f65008b8e626d63
Top:    799817db26572836f175d32bdd60ba07cd85538c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:35:42 +0100

Refresh of move-c-fix-incorrect-usage-of

---

diff --git a/src/move.c b/src/move.c
index d785823..cf2ab88 100644
--- a/src/move.c
+++ b/src/move.c
@@ -910,7 +910,7 @@ move_all_thermals (int old_index, int new_index)
 static int
 LastLayerInComponentGroup (int layer)
 {
-  int cgroup = GetLayerGroupNumberByNumber(max_group + COMPONENT_LAYER);
+  int cgroup = GetLayerGroupNumberByNumber(component_silk_layer);
   int lgroup = GetLayerGroupNumberByNumber(layer);
   if (cgroup == lgroup
       && PCB->LayerGroups.Number[lgroup] == 2)
@@ -921,7 +921,7 @@ LastLayerInComponentGroup (int layer)
 static int
 LastLayerInSolderGroup (int layer)
 {
-  int sgroup = GetLayerGroupNumberByNumber(max_group + SOLDER_LAYER);
+  int sgroup = GetLayerGroupNumberByNumber(solder_silk_layer);
   int lgroup = GetLayerGroupNumberByNumber(layer);
   if (sgroup == lgroup
       && PCB->LayerGroups.Number[lgroup] == 2)
